config.yaml

```diff
@@ -258,6 +258,15 @@ restservice:
   extra_env: {}
 
 nginx:
+  # Number of nginx worker processes to have.
+  worker_processes: 1
+
+  # Total number of open file descriptors to allow per nginx worker.
+  max_open_fds: 10240
```

Comment:
How did you arrive at this number? Is it the underlying default?

Reply:
Yeah, I think it's probably best to leave it auto-calculated for the moment, unless we have a specific requirement. Otherwise, someone is going to typo it at some point and then we'll have really 'fun' errors to troubleshoot.